doc/source/admin/authentication.md

```diff
@@ -0,0 +1,210 @@
+filter to disable registration for users using known disposable email provider. 
+
+How to set up this config is presented here.
+
+*Note: SQLite database is not supported with this feature. Please use PostgreSQL.*
+
+### Account activation feature
+
+In the Galaxy config file **config/galaxy.ini** there is the user activation setting that you have to turn on.
+
+```
+user_activation_on = True
+```
+
+
+There is also the option for tracking jobs in database that is required to be turned on for the account activation to be effective. By default it is off.
+
+```
+track_jobs_in_database = True
+```
+
+
+After you turn on both of these every user that will try to register after this configuration file takes effect will have the verification email sent to the email address provided. Unless the Grace period (see below) is set, the user won't be able to login before the verification happens.
+
+Furthermore in order for this to work correctly smtp server and admin email should be set:
+```
+#smtp_server = some.server.edu:587
+#smtp_username = example_username
+#smtp_password = example_passsword
+#activation_email = activation-noreply@example.com
+#error_email_to = admin@example.com
+```
+
+Smtp server takes care of the email sending and the activation_email email is used as the *From* address in the verification email. Furthermore the error_email_to is being shown to the user if the Galaxy detects its own misconfiguration.
+
+You can also set the instance_resource_url which is shown in the activation emails so you can point users to your wiki or other materials.
+```
+instance_resource_url = http://wiki.galaxyproject.org/
+```
+
+
+The final activation email looks like this:
+
+```
+Hello <user_name>,
+
+In order to complete the activation process for <user_email> begun on <date> at <hostname>, please click on the following link to verify your account:
+
+test.galaxyproject.org/activate?activation_token=46701ecdbbf2a79a7348ddae33062774edadef59&email=example%40example.com
+
+By clicking on the above link and opening a Galaxy account you are also confirming that you have read and agreed to Galaxy's Terms and Conditions for use of this service (<link_to_terms_config>). This includes a quota limit of one account per user. Attempts to subvert this limit by creating multiple accounts or through any other method may result in termination of all associated accounts and data.
+
+Please contact us if you need help with your account at: <error_email_to_config>. You can also browse resources available at: <instance_resources_url_config>.
+
+More about the Galaxy Project can be found at galaxyproject.org
+
+Your Galaxy Team
+
+```
+
+### Changing email address
+
+If activated user changes email address in user settings his/her account will be deactivated. A new activation link will be sent and user is required to visit it to activate the account again.
```

Comment:
If an activated user changes email address in user settings, his/her account will be deactivated. A new activation link will be sent and the user will have to visit it to activate the account again.